public/css/showoff.css

```diff
@@ -135,6 +135,11 @@ pre code {
   }
 }
 
+.ui-button:active {
```

Comment:
did you validate that this doesn't negatively impact other uses of the button class?